src/ColorPicker/index.jsx

```diff
@@ -49,7 +49,7 @@ function ColorPicker({
               `pgn__color-picker-${size}`,
             )}
             style={{
-              ...(color && hexValid ? { background: `${color}` } : {}),
+              ...(color && hexValid ? { background: `${color} !important` } : {}),
```

Comment:
[curious] Why is `!important` needed here? My understanding is that inline styles through the `style` attribute should already have preference. Needing to use `!important` is generally a code smell (though not _always_).